src/convertToObject.js

```diff
@@ -13,6 +13,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const strToArr = sourceString.split(';').filter(item => item.length > 4);
```

Comment:
Why 4? Use more obvious condition.
Think about how to get rid of unnecessary spaces.